QUEST_LV_0100.tsv

```diff
@@ -297,7 +297,7 @@ QUEST_LV_0100_20150317_000296	$Stamina is consumed when you move.{nl}You can rec
 QUEST_LV_0100_20150317_000297	$I heard news from my men and was becoming worried.{nl}All is well if it ended up good.
 QUEST_LV_0100_20150317_000298	$Klaipeda Resident
 QUEST_LV_0100_20150317_000299	$)} {nl}When do you think the Goddess is coming back?{nl}Even if I ask the Revelators, all they say is that they must go to Klaipeda...
-QUEST_LV_0100_20150317_000300	$Are you a Revelator?{nl}Didn't the Goddess appeared in your dream?{nl}I've heard so from the soldiers.
+QUEST_LV_0100_20150317_000300	Are you a Revelator?{nl}Didn't a Goddess appear to you in a dream?{nl}I heard so from the soldiers.
```

Comment:
She's asking for confirmation.

So another way of saying this is : A goddess appeared in your dreams, right. 

I'll change the a to your, not sure why I changed it to a o-o.